src/test/java/org/jacp/competitionservice/AbstractContainerBaseTest.java

```diff
@@ -0,0 +1,26 @@
+package org.jacp.competitionservice;
+
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.HostConfig;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.MountableFile;
+
+public abstract class AbstractContainerBaseTest {
```

Comment:
Do you think we can give a name that will make it clear how this class relates to Postgre?

Reply:
Fixed